health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/CreateQuestion.js

```
@@ -340,175 +340,180 @@ const CreateQuestion = ({ onSelect, className, level = 1, initialQuestionData, p
-                        // style={{height:"1rem", alignItems:"center", paddingBottom:"0.5rem"}}
-                        key={field.key}
-                        mainClassName={"checkboxOptionVariant"}
-                        // disabled={optionDependency ? true : false}
-                        label={t("REQUIRED")}
-                        checked={field?.isRequired}
-                        // onChange={handleRequiredField(field.id)}
-                        onChange={() => handleRequiredField(field.id)}
-                        // isLabelFirst={true}
-                        index={field.key}
-                      />
+            <div>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)